lib/active_force/active_query.rb

```diff
@@ -118,6 +120,83 @@ def order *args
 
     private
 
+    def build_includes(relation)
+      association = sobject.associations[relation]
+      fields Association::EagerLoadProjectionBuilder.build(association)
+      association_mapping[association.sfdc_association_field.downcase] = association.relation_name
+    end
+
+    def build_hash_includes(relation)
+      relation.each do |key, value|
+        association = sobject.associations[key]
+        association_name = association.class.name.split('::').last
```

Comment:
```suggestion
        association_name = association.class.name.demodulize
```
ActiveSupport has something for this already